src/citrine/resources/material_run.py

```diff
@@ -109,8 +109,7 @@ def get_type(cls) -> Type[MaterialRun]:
         """Return the resource type in the collection."""
         return MaterialRun
 
-    def get_history(self, *, id: Union[str, UUID, LinkByUID, MaterialRun],
-                    scope: Optional[str] = None) -> Type[MaterialRun]:
+    def get_history(self, *, id: Union[str, UUID, LinkByUID, MaterialRun]) -> Type[MaterialRun]:
```

Comment:
Not today probably, but this should _definitely_ allow for a positional argument.

Reply:
Yes, other similar functions don't require a keyword arg. And that's an easy change